librecad/src/lib/information/rs_information.cpp

```diff
@@ -109,12 +110,12 @@ bool RS_Information::isTrimmable(RS_Entity* e1, RS_Entity* e2) {
                     e1->getParent()==e2->getParent()) {
 
                 // in the same polyline
-                RS_EntityContainer* pl = e1->getParent();
+                RS_Polyline* pl = (RS_Polyline *)e1->getParent();
                 int idx1 = pl->findEntity(e1);
                 int idx2 = pl->findEntity(e2);
                 RS_DEBUG->print("RS_Information::isTrimmable: "
                                 "idx1: %d, idx2: %d", idx1, idx2);
-                if (abs(idx1-idx2)==1 || abs(idx1-idx2)==int(pl->count()-1)) {
+                if (abs(idx1-idx2)==1 || (pl->isClosed() && abs(idx1-idx2)==int(pl->count()-1))) {
```

Comment:
Not sure - but at least that's how it worked before, and I haven't noticed any other bugs related to that. I've only changed it to ignore the 'first and last index' (wraparound) case for open polylines, and it at least seems to work as expected.